workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layouts/1_home/page-definition.json

```diff
@@ -332,7 +338,7 @@
 																			},
 																			"url": {
 																				"value_i18n": {
-																					"en_US": "[$DOCUMENT_URL:/site-initializer/documents/group/icons/cloud.svg$]"
+																					"en_US": "/documents/d/liferay-learn/cloud-svg"
```

Comment:
Why did you remove $DOCUMENT_URL:/site-initializer/?
Can you keep it?
I think is better to do a find replace.

Reply:
It was the autogenerated JSON exported file. 
Sure ! I'll re-add it